tools/accuracy_checker/openvino/tools/accuracy_checker/postprocessor/resize_prediction_boxes.py

```diff
@@ -59,6 +63,14 @@ def process_image_with_metadata(self, annotation, prediction, image_metadata=Non
             w = self.image_size[1] / input_w
             h = self.image_size[0] / input_h
 
+        if self.unpadding:
+            input_h, input_w, _ = image_metadata.get('image_info', self.image_size)
+            _, _, i_h, i_w = image_metadata.get('input_shape')['input']
```

Comment:
please take into account that model can have NHWC layout

Reply:
Ok, I think that this code should be refactored. Moreover, there are 2 general ways to preprocess image for this model:
1. Fit image along larger side to model's window size and pad along smaller side.
2. Fit image along smaller side to model's window size  and crop along larger side, for example, central area of image.
In both ways geometric transformations in image metadata need to be properly handled by postprocessor to restore prediction box coordinates into coordinate space of source image.

Reply:
Fixed